packages/api-key-stamper/src/utils.ts

```diff
@@ -9,12 +15,39 @@ export function convertTurnkeyApiKeyToJwk(input: {
 
   const jwk = pointDecode(hexStringToUint8Array(compressedPublicKeyHex));
```

Comment:
If x and y aren't padded for whatever reason, let's fix this inside of `pointDecode` so we don't have to extract, pad, and then put things "back" in the JWK:
https://github.com/tkhq/sdk/blob/08e2b8c30848498ed0850aa1e1ede67775b001e5/packages/api-key-stamper/src/tink/elliptic_curves.ts#L154-L155